tests/test_batch.py

```diff
@@ -1,92 +1,106 @@
-# Unit tests related to 'Batch' (https://www.easypost.com/docs/api#batches).
-
-from time import sleep
-
 import pytest
 
 import easypost
 
 
 @pytest.mark.vcr()
-def test_batch_create_and_buy(vcr):
-    # We create Address and Parcel objects. We then try to create a Batch containing a shipment.
-    # Finally, we assert on saved and returned data.
-
-    # from address and parcel don't change
-    from_address = easypost.Address.create(
-        name="Simpler Postage Inc.",
-        street1="388 Townsend St",
-        street2="Apt 20",
-        city="San Francisco",
-        state="CA",
-        zip="94107",
-        phone="415-456-7890",
+def test_batch_create(one_call_buy_shipment):
+    batch = easypost.Batch.create(shipments=[one_call_buy_shipment])
+
+    assert isinstance(batch, easypost.Batch)
+    assert str.startswith(batch.id, "batch_")
+    assert batch.shipments is not None
+
+
+@pytest.mark.vcr()
+def test_batch_retrieve(one_call_buy_shipment):
+    batch = easypost.Batch.create(shipments=[one_call_buy_shipment])
+
+    retrieved_batch = easypost.Batch.retrieve(batch.id)
+
+    assert isinstance(batch, easypost.Batch)
+    assert retrieved_batch.id == batch.id
+    # status changes between creation and retrieval, so we can't compare the whole object
```

Comment:
A comment after the code its referencing is confusing. Please always have comments either come right before the line they are referencing or on the same line (depending on length and context.)